libs/SmartStore/build.gradle

```diff
@@ -4,6 +4,10 @@ apply plugin: 'com.android.library'
 dependencies {
   compile project(':libs:SalesforceSDK')
   compile 'net.zetetic:android-database-sqlcipher:3.5.2'
+  androidTestCompile 'org.mockito:mockito-core:1.10.9'
```

Comment:
Why do we have these extra dependencies? I don't think we want to add these dependencies.